### PR TITLE
MGMT-3088 push images to quay.io when we build from branch for instal…

### PR DESCRIPTION
**Pull request by oshercc**
…ler & agent

### CODE REVIEW

**Review by gamli75 — APPROVED** [review]

**Review by YuviGold — APPROVED** [review]
LGTM

File: Jenkinsfile [hunk 1]
```diff
@@ -3,6 +3,10 @@ String cron_string = BRANCH_NAME == "master" ? "@daily" : ""
 pipeline {
   environment {
         AGENT_IMAGE = 'quay.io/ocpmetal/assisted-installer-agent'
```
Comment (YuviGold) [inline Jenkinsfile:5]: You can remove this one, it isn't used.

**Review by YuviGold — APPROVED** [review]

**Pull request merged**